### ISSUE TITLE
New Tab Does Not Open After Clicking on Link.dox

### ISSUE THREAD

**Issue by jill-iterators** [CityOfBoston/Iterators#154]
Screengrab/videos (insert file names here)
New Tab Does Not Open After Clicking on Link.mp4

Link
https://d8-stg.boston.gov/departments/emergency-medical-services/cpr-training

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.5 >Chrome 84

Trusted Tester Test
N/a

Brief Description
When clicking on a new link, a new tab should open so that you are still on the Boston.gov website.

Steps to reproduce
1)	00:00, Go to the site, using link provided above. 
2)	00:15, Click on the link titled: Learn More About Hands-Only CPR
3)	00:18, A new window does not open, instead the user loses connection with City of Boston and is now on the American Heart Association site. 
a)	This is a bug. If Iterators were only doing functional testing, this would be flagged as a functional bug. A best practice is to have a new window open when clicking on an external  link. In addition, there are many places on Boston.gov, where a new window would open. Therefore, it’s confusing for the individual using the site to get two different results. For consistency, they should be all one-way.
b)	Given, Iterators is completing Accessibility testing, this is not specifically specified under Sec508 conformance testing. However, a user of the site, who is visually impaired as an example, will be thrown, and apt to lose Focus after clicking on a new the link and losing access to Boston.gov.

Desired Result
•	When clicking on a new link, a new tab opens up and the user is not kicked off of the Boston.gov website.

Actual Result 
•	Unable to stay on the Boston.gov website when clicking on an external link, so that I don’t become confused and lose focus.

There are many instances of where the user will lose access to Boston.gov as a new window does not open for access to the new page selected. A listing of all the links would need to be produced to complete this.

**Comment by subaha-cob** [comment]
Test in UAT: https://d8-uat.boston.gov/departments/emergency-medical-services/cpr-training

**Comment by jill-iterators** [comment]
Working as expected. Will close the ticket.
